server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndex.java

```diff
@@ -286,9 +286,7 @@ private static AbstractAggregationBuilder createQualityGateFacet() {
       BoolQueryBuilder metricFilters = boolQuery();
       entry.getValue()
         .stream()
-        .map(criterion -> nestedQuery(FIELD_MEASURES, boolQuery()
-          .filter(termQuery(FIELD_MEASURES_KEY, criterion.getMetricKey()))
-          .filter(toValueQuery(criterion))))
+        .map(criterion -> toQuery(criterion))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612)